dist/testbed-support/makefile.iotlab.single.inc.mk

```diff
@@ -0,0 +1,190 @@
+# Use a single iot-lab node with RIOT
+# ===================================
+#
+# Supported:
+#  * flash
+#  * reset
+#  * term
+#
+# Tested on m3/a8-m3/wsn430/samr21/arduino-zero nodes
+#
+# It can be run:
+# * From your computer by setting IOTLAB_NODE to the full url like
+#   m3-380.grenoble.iot-lab.info or a8-1.grenoble.iot-lab.info
+# * From IoT-LAB frontend by setting IOTLAB_NODE to the short url like
+#   m3-380 or a8-1
+#
+# # Usage
+#
+#   make BOARD=iotlab-m3 IOTLAB_NODE=m3-380.grenoble.iot-lab.info flash term
+#
+# It is the user responsability to start an experiment beforehand
+#
+# If the user has multiple running experiments, the one to use must configured
+# with the `IOTLAB_EXP_ID` setting
+#
+# Prerequisites
+# -------------
+#
+# * Install iotlab-cli-tools: https://github.com/iot-lab/cli-tools
+# * Install iotlab-ssh-cli-tools: https://github.com/iot-lab/ssh-cli-tools
+#   * Can be done using given `requirements.txt`
+# * Configured your account with 'iotlab-auth'
+# * If using an A8 node from a frontend:
+#   * Register the frontend public key in your ssh keys list on your profile.
+#
+# Additional for A8 nodes
+#
+# * Install iotlab-ssh-cli-tools https://github.com/iot-lab/ssh-cli-tools
```

Comment:
This is already the second point of this list

Reply:
Oh I should remove it from the first list then, I thought I missed it.